defaults/main.yml

```diff
@@ -527,6 +527,12 @@ gitlab_runner__cache_secret_key: ''
 gitlab_runner__cache_bucket_name: ''
 
                                                                    # ]]]
+# .. envvar:: gitlab_runner__cache_bucket_location [[[
+#
+# Cache bucket location
+gitlab_runner__cache_bucket_location: 'us-east-1'
```

Comment:
Sounds good, although I would set that as an empty string and provide a link to the [AWS Availability Zone documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html). That way the user will need to check what zones are available and set one, hopefully picking the right one. It's also how the rest of the variables in the role related to AWS are specified.